docs/HEROKU_DEPLOY.md

```diff
@@ -0,0 +1,8 @@
+# Instructions for after you click - deploy to heroku
```

Comment:
this is a random thought from Scott's documentation link a while ago.  I wonder if we should prefix all our doc file names with e.g. HOWTO_ or REFERENCE_ etc.  thoughts?